src/providers/oauth-token-provider.js

```diff
@@ -0,0 +1,111 @@
+
+/**
+ * Module dependencies.
+ */
+
+import angular from 'angular';
+
+/**
+ * Token provider.
+ */
+
+function OAuthTokenProvider() {
+  var config = {
+    name: 'token',
+    options: {
+      secure: true
+    }
+  };
+
+  /**
+   * Configure.
+   *
+   * @param {object} params - An `object` of params to extend.
+   */
+
+  this.configure = function(params) {
+    if (params instanceof Object) {
+      angular.extend(config, params);
+    } else {
+      throw 'Invalid argument: `config` must be an `Object`.';
+    }
+
+    return config;
+  };
+
+  /**
+   * OAuthToken service.
+   *
+   * @ngInject
+   */
+
+  this.$get = function(ipCookie) {
+    class OAuthToken {
+
+      /**
+       * Get authorizationHeader.
+       */
+
+      get authorizationHeader() {
+        // return (this.tokenType && this.accessToken) ? `${this.tokenType} ${this.accessToken}` : undefined;
```

Comment:
It is case sensitive as defined in [RFC 6750](https://tools.ietf.org/html/rfc6750#section-2.1).